jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessRuntimeDataServiceImpl.java

```diff
@@ -98,51 +97,28 @@ public ProcessInstanceSummary getProcessInstance(String serverTemplateId,
-        PrettyTime prettyDateFormatter = new PrettyTime();
+        ProcessServicesClient processServicesClient = getClient(serverTemplateId,
+                                                                ProcessServicesClient.class);
 
-        for (int i = processInstanceHistory.size() - 1; i >= 0; i--) {
-            NodeInstance nis = processInstanceHistory.get(i);
-
-            if (nis.getNodeType().equals("HumanTaskNode")) {
-                logs.add(new RuntimeLogSummary(nis.getId(),
-                                               prettyDateFormatter.format(nis.getDate()),
-                                               "Task '" + nis.getName() + "' was created",
-                                               "System"));
-                for (TaskEventInstance te : allTaskEventsByProcessInstanceId) {
-                    if (te.getWorkItemId() != null && te.getWorkItemId().equals(nis.getId()) &&
-                            (te.getType().equals("CLAIMED") || te.getType().equals("RELEASED") || te.getType().equals("COMPLETED"))) {
-                        logs.add(new RuntimeLogSummary(nis.getId(),
-                                                       "- " + prettyDateFormatter.format(te.getLogTime()),
-                                                       "Task '" + nis.getName() +
-                                                               "' was " + te.getType().toLowerCase() + " by user " + te.getUserId(),
-                                                       "Human"));
-                    }
-                }
-            } else if (nis.getNodeType().equals("StartNode")) {
-                logs.add(new RuntimeLogSummary(nis.getId(),
-                                               prettyDateFormatter.format(nis.getDate()),
-                                               "Process '" + processName + "' was created",
-                                               "Human"));
-            } else if (nis.getNodeType().equals("EndNode")) {
-                logs.add(new RuntimeLogSummary(nis.getId(),
-                                               prettyDateFormatter.format(nis.getDate()),
-                                               "Process '" + processName + "' was completed",
-                                               "System"));
-            }
-        }
+        List<NodeInstance> processInstanceHistory = processServicesClient.findNodeInstances(deploymentId,
+                                                                                            processInstanceId,
+                                                                                            0,
+                                                                                            100);
+        List<RuntimeLogSummary> logs = new ArrayList<>();
+        PrettyTime prettyDateFormatter = new PrettyTime();
```

Comment:
We can't really format the date in here because it won't consider the user locale, we should try to reuse the Moment API at the presenter level for this.